{"name": "TC_001 : Validating the CredKart URL", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <testCases.test_CredKart_User_Profile_001.Test_User_Profile_Class_001 object at 0x0000016334C64440>\n\n    @allure.description(\"Validating the CredKart URL\")\n    @allure.title(\"TC_001 : Validating the CredKart URL\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    @allure.story(\"Story : Week3\")\n    @allure.epic(\"Epic : 2nd level\")\n    @pytest.mark.sanity\n    @pytest.mark.group1\n    @pytest.mark.flaky(reruns=0, reruns_delay=2)\n    @pytest.mark.dependency(name = \"test_verify_CredKart_Url_001\")\n    def test_verify_CredKart_Url_001(self):\n        self.log.info(\"Testcase test_verify_CredKart_Url_001 is started\")\n        self.driver.get(self.homepage_url)\n        self.log.info(\"Opening Browser\")\n        self.log.info(f\"Going to CredKart URL ->{self.homepage_url}\")\n        self.log.info(\"Checking Page Title\")\n        if self.driver.title == \"CredKart1\":\n            allure.attach.file(\".\\\\Screenshots\\\\test_verify_CredKart_Url_001_Fail.png\", name = \"test_verify_CredKart_Url_001_Fail.png\", attachment_type = allure.attachment_type.PNG)\n            self.log.info(f\"Page Title:'{self.driver.title}' is  matches with expected title\")\n            self.log.info(\"Taking Screenshot\")\n            additional_utilities_class.take_screenshot(self.driver,\"test_verify_CredKart_Url_001\", \"Pass\" )\n            self.log.info(\"Testcase test_verify_CredKart_Url_001 is passed\")\n            self.log.info(\"Testcase test_verify_CredKart_Url_001 is completed\\n\")\n            assert True\n        else:\n            self.log.info(\"Taking Screenshot\")\n            #self.driver.save_screenshot(\".\\\\Screenshots\\\\test_verify_CredKart_Url_001_fail.png\")\n            additional_utilities_class.take_screenshot(self.driver, \"test_verify_CredKart_Url_001\", \"Fail\")\n            self.log.info(\"Testcase test_verify_CredKart_Url_001 is failed\")\n            self.log.info(\"Testcase test_verify_CredKart_Url_001 is completed\\n\")\n>           assert False\nE           assert False\n\ntestCases\\test_CredKart_User_Profile_001.py:61: AssertionError"}, "description": "Validating the CredKart URL", "attachments": [{"name": "log", "source": "31416df4-efc5-48fa-8cfc-b32a2118e56f-attachment.txt", "type": "text/plain"}], "start": 1733711366607, "stop": 1733711369536, "uuid": "162614b6-af84-4fde-9203-4b293c288171", "historyId": "ca43a64682bf5138fdf2b7b9d7bf7ebd", "testCaseId": "ca43a64682bf5138fdf2b7b9d7bf7ebd", "fullName": "testCases.test_CredKart_User_Profile_001.Test_User_Profile_Class_001#test_verify_CredKart_Url_001", "labels": [{"name": "story", "value": "Story : Week3"}, {"name": "severity", "value": "critical"}, {"name": "epic", "value": "Epic : 2nd level"}, {"name": "tag", "value": "dependency(name='test_verify_CredKart_Url_001')"}, {"name": "tag", "value": "flaky(reruns=0, reruns_delay=2)"}, {"name": "tag", "value": "group1"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('driver_setup')"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_CredKart_User_Profile_001"}, {"name": "subSuite", "value": "Test_User_Profile_Class_001"}, {"name": "host", "value": "Credence-Pc"}, {"name": "thread", "value": "31120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_CredKart_User_Profile_001"}]}